pull_request_template.md

```diff
@@ -33,25 +34,11 @@
 
 <!-- Describe the steps to test the changes that this PR introduces; or explain the reason why it is not possible to test the changes -->
 
-## Assignee
-
-<!-- Add mentions to the PR author and also to reviewers who tested the changes in UAT -->
-
-## Screenshots
-
-<!-- Add screenshots, link to Loom videos, or link to wireframes/mockups if they are relevant; e.g.: for frontend work -->
-
 ## Checklist
 
 - Manual tests
   - [ ] Changes were tested locally by the author
-  - [ ] Changes were tested on staging by the author
```

Comment:
It still should say UAT here for recruit and staging for onboard, so depends on which repos will use this

Reply:
recruit is automatically overwritten by it's repos pull request template. onboard also has it's own PR template